vms/rpcchainvm/vm_server.go

```diff
@@ -405,10 +399,7 @@ func (vm *VMServer) Connected(ctx context.Context, req *vmpb.ConnectedRequest) (
 }
 
 func (vm *VMServer) Disconnected(ctx context.Context, req *vmpb.DisconnectedRequest) (*emptypb.Empty, error) {
-	nodeID, err := ids.ToNodeID(req.NodeId)
-	if err != nil {
-		return nil, err
-	}
+	nodeID := ids.NodeIDFromBytes(req.NodeId, len(req.NodeId))
```

Comment:
This should error if the bytes aren't 32 or 20

Reply:
Done